package.json

```diff
@@ -79,7 +79,7 @@
     "css-loader": "^5.0.1",
     "d3": "^4.13.0",
     "debug": "^4.2.0",
-    "electron": "12.0.2",
+    "electron": "12.2.3",
```

Comment:
Huh? https://github.com/balena-io/etcher/blob/master/package.json#L72 is already using `"electron": "12.2.3"` ?